web/permissions/permissions.go

```diff
@@ -278,6 +295,17 @@ func patchPermission(getPerms getPermsFunc, paramName string) echo.HandlerFunc {
 			}
 		}
 
+		// Handle metadata
+		// If the metadata has been given in the body request, just apply it to
+		// the patch
+		if patch.Metadata != nil {
+			toPatch.Metadata = patch.Metadata
+		} else if toPatch.Metadata != nil { // No metadata given in the request, but it does exist in the database: update it
+			updatedMD := toPatch.Metadata.Clone()
+			updatedMD.Update()
+			toPatch.Metadata = &updatedMD
+		}
+
```

Comment:
We should also look at the token to know which app has made the update to fill `UpdatedByApps`